packages/babel-plugin-transform-es2015-modules-amd/src/index.js

```diff
@@ -14,34 +14,22 @@ const buildWrapper = template(`
   })
 `);
 
-export default function({ types: t }) {
+export default function({ types: t }, options) {
+  const { allowTopLevelThis, loose, noInterop, strict, strictMode } = options;
+  const ref = { allowTopLevelThis, loose, noInterop, strict, strictMode };
```

Comment:
`ref` could be renamed to smth like `optionsCopy`, is there even need to copy it? is there danger that it gets mutated?

Reply:
Actually I thin the same as @existentialism here - https://github.com/babel/babel/pull/6381#discussion_r142783867